package.json

```diff
@@ -10,14 +10,15 @@
     "@actions/exec": "^1.0.3",
     "@actions/github": "^2.1.1",
     "@types/jest": "^26.0.0",
+    "@vercel/ncc": "^0.36.1",
     "got": "^11.3.0",
-    "lodash.get": "^4.4.2"
+    "lodash.get": "^4.4.2",
+    "typescript": "^5.0.3"
```

Comment:
These deps should go into devDeps, no?

Reply:
@Steveantor what do you think

Reply:
typescript is required for the build process

Reply:
If it's build-related, this should then be a dev dep.

Reply:
I agree, needs to be moved to dev-deps in that sense.